app/components/myissues.jsx

```diff
@@ -0,0 +1,161 @@
+var React = require("react");
+var getJSON = require("./getJSON.jsx");
+var { auth, AuthMixin } = require("./auth.jsx");
+var APIServer = "/api";
+var Labels = require("./labels.jsx");
+
+var IssueCard = React.createClass({
+  getInitialState: function() {
+    return {
+    };
+  },
+
+  getShortRepoString: function (fullUrl) {
+    var s = fullUrl;
+    s = s.replace("https://github.com/", "");
+    var i = s.indexOf("/");
+    s = s.slice(i+1);
+    i = s.indexOf("/");
+    s = s.slice(0, i);
+    return s;
+  },
+
+  render: function() {
+    var data = this.props.data;
+    var repo = this.getShortRepoString(data.html_url);
+    return (
+      <li>
+          <a href={data.html_url} target="_blank">
+              <h4>{data.title} <br/><small>{repo}</small></h4>
+              <Labels labels={data.labels} minimal='true' />
+          </a>
+      </li>
+      );
+  }
+});
+
+var ShowMyIssues = React.createClass({
+  getInitialState: function() {
+    return {
+      issuegroups: []
+    };
+  },
+  componentDidMount: function() {
+    var self = this;
+    // Assigned issues
+    getJSON(APIServer + "/myissues/assigned",
+      function(data) {
+        if (self.isMounted()) {
+          var issues = { title: 'Assigned',
+                        issues: data.issues}
+          var newState = self.state.issuegroups;
+          newState.push(issues)
+          self.setState(newState);
```

Comment:
Thanks for the explanation. That does make sense now.

I was totally guessing how to use setState as I was writing that :stuck_out_tongue_winking_eye:

I'll push the latest commit right now.